### PR TITLE
feat: PATRON2020-339 redesign sensors and notifications lists

### PR DESCRIPTION
**Pull request by kamiljalowiczor**
- Lists now have corresponding sensors' icons instead of a color,
- Each item of these lists elevates when clicked on,
- Added unknown sensor icon,
- Added temperature sensor icons,
- Sensors on map now have a thin, black border (except for light sensors)

Sensor list before: 
before

Sensor list Now:
now

### CODE REVIEW

File: frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphics/TemperatureSensorValue.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const useStyles = makeStyles((props) => ({
   })
 }))
 
-export default function TemperatureSensorInner ({ temperature }) {
+export default function TemperatureSensorInner ({ temperature = 15 }) {
```
Comment (jwieckowski) [inline frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphics/TemperatureSensorValue.js:13]: why do we need this default assignment here?

File: frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphics/WindowBlindsSensorIcon.js [hunk 2]
```diff
@@ -9,20 +9,20 @@ const useStyles = makeStyles((props) => ({
...
   blind: props => ({
-    fontSize: '1.55vh',
+    fontSize: '1.6vh',
     clipPath: `polygon(0 0, 100% 0, 100% ${props.position}%, 0 ${props.position}%)`,
     color: 'rgba(0, 0, 0, 0.65)'
   })
 }))
 
-export default function WindowBlindsSensorInner ({ position }) {
+export default function WindowBlindsSensorInner ({ position = 50 }) {
```
Comment (jwieckowski) [inline frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphics/WindowBlindsSensorIcon.js:25]: same question here

File: frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphics/TemperatureSensorValue.js (hunk 1 above)
Comment (kamiljalowiczor) [reply frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphics/TemperatureSensorValue.js:13]: We don't pass any values when displaying a notifications list, but they are needed to display some of the icons. Default assignment solves this issue.

File: frontend/src/components/Dashboard/SensorsList/Item.jsx [hunk 3]
```diff
@@ -153,54 +169,62 @@ const Item = ({ sensorData, isOnMap, handleRemoveClick, expanded, handleChangeEx
...
-            }
-          />
-          {drawItemInfo(type, sensorData, classes, handleRemoveClick)}
-        </ListItem>
-      </ExpansionPanelSummary>
-      {expansionPanelDetails}
-    </ExpansionPanel>
+          <ListItem
+            id={`sensor${id}`}
+            onClick={(e) => {
```
Comment (makeros) [inline frontend/src/components/Dashboard/SensorsList/Item.jsx:187]: Please move bigger functions out of the jsx part :)

File: frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphics/TemperatureSensorIcon.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles((props) => ({
+  size: {
+    fontSize: '2.4vh'
+  }
+}))
+
+export default function TemperatureSensorIcon ({ temperature = 15 }) {
```
Comment (makeros) [inline frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphics/TemperatureSensorIcon.js:13]: Is somewhere a test for this functionality?

Comment (kamiljalowiczor) [reply frontend/src/components/Dashboard/SmartHomeMap/Map/Sensor/SensorGraphics/TemperatureSensorIcon.js:13]: oh right i forgot about this sensor

**Pull request merged**